docs/support-bundle-guide.md

```diff
@@ -0,0 +1,247 @@
+  antrea-controller.conf: |
+    featureGates:
+    # Enable collecting support bundle files with SupportBundleCollection CRD.
+    SupportBundleCollection: true
+```
+
+A single namespace (e.g., default) is created for saving the secrets which are
+used to access the support bundle file server, and the permission to read secrets
+in this namespace is given to antrea-controller by modifying and applying the
+[RBAC file](/build/yamls/externalnode/support-bundle-collection-rbac.yml).
+
+```yaml
+...
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: antrea-read-secrets
+  namespace: default  # Change the Namespace to where the Secret for file server's authentication credential is created.
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: antrea-secret-reader
+subjects:
+  - kind: ServiceAccount
+    name: antrea-controller
+    namespace: kube-system
+```
+
+## The SupportBundleCollection CRD
+
+SupportBundleCollection CR is introduced to supplement the `antctl` command with
+three reinforcements:
+
+1. Allow users to collect support bundle files on ExternalNodes.
+2. Upload all the support bundle files into a user-provided SFTP Server.
+3. Support tracking status of a SupportBundleCollection CR.
+
+Details are provided in the next section.
+
+## Use case example
+
+### Running antctl commands
+
+Please refer to the corresponding [antctl user guide section](antctl.md#collecting-support-information).
+Note: `antctl supportbundle` only works on collecting the support bundles from Antrea Controller and
+Antrea Agent which is running on K8s Node, it does not work for the Agent on an external Node.
+
+### Applying SupportBundleCollection CRD
+
+In this section, we will create two SupportBundleCollections for Kubernetes Nodes and ExternalNodes.
+Assume we have a cluster with Nodes named "kube-worker" and "kube-master". In addition, we have
+set up an ExternalNode as `vm` in the `vm-ns` Namespace by following the instruction
+in [Install ExternalNode on VM](external-node.md#install-antrea-agent-on-vm).
+
+A Secret needs to be created in advance with the username and password of the SFTP Server.
+The Secret will be used in `authSecret` in the following YAML examples.
+
+```bash
+kubectl create secret generic support-bundle-secret  --from-literal=username='your-sftp-username'  --from-literal=password='your-sftp-password'
+```
+
+Then we could apply the following YAML files. The first one is to collect support bundle
+on Kubernetes Nodes "kube-worker" and "kube-master". The second one is to collect support
+bundle on ExternalNode "vm".
+
+```bash
+cat << EOF | kubectl apply -f -
+kind: SupportBundleCollection
+metadata:
+  name: collect-support-bundle-for-kube-nodes
+spec:
+  nodes:
+    nodeNames: # Empty list means all the Nodes
+      - kube-master
+      - kube-worker
+    matchLabels:
+      app: nginx   # Collect support bundle files from the Nodes which have been labeled as "app: nginx".
```

Comment:
Should we comment it out? Or you should add description about Nodes with the nginx label.